apps/openchallenges/mcp-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+server:
+  port: 8887
```

Comment:
- Register this port in `.devcontainer/devcontainer.json`. VS Code will auto-forward the port starting from the next rebuild of the dev container. It will also attach the name `openchallenges-mcp-server` to the port.
- Add an entry for `openchallenges-mcp-server` in `tools/configure-hostnames.sh`. This will allow to do `curl -i http://openchallenges-mcp-server:{port}`. This will also enable to develop the MCP server using its development server with other components that are deployed with Docker in the future.